core/server/models/base/index.js

```diff
@@ -302,17 +302,12 @@ ghostBookshelf.Model = ghostBookshelf.Model.extend({
 
     /**
      * please use these static definitions when comparing id's
-     * we keep type number, because we have too many check's where we rely on
+     * we keep type Number, because we have too many check's where we rely on Number
      * context.user ? true : false (if context.user is 0 as number, this condition is false)
      */
     internalUser: 1,
-    ownerUser: 1,
     externalUser: 0,
 
-    isOwnerUser: function isOwnerUser(id) {
```

Comment:
This fn was invented to have an easy, centralised comparison for the owner user, but transferring the ownership can result in an owner user with the id being not '1'. The function was added when we worked on OAuth, not used anyway right now.